src/index.html

```diff
@@ -17,16 +17,27 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-container"
+      data-qa="big"
+    >
+      <input
+        class="search-bar search-bar--big"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
```

Comment:
The attributes in the `input` element of the first `form` should start on a new line with 2-space indentation related to the tag. Also, ensure that the line of code does not exceed 80 characters.